### PR TITLE
fix: Add basic integration tests

### PR DESCRIPTION
**Pull request by wtrocki**
Current repo settings is allowing to introduce changes that do not work quite easly. 
Renovate merges changes automatically but there are no single test to check if anything in the repository will compile and produce result. 

This change tries to evaluate on existing template to see if codegen and generate works. We can follow with graphql-inspector commands easily to have full end to end test

### CODE REVIEW

**Comment by wtrocki** [comment]
Locally fails with:

```
  build failed with Error: Command failed with exit code 1: yarn graphql generate
  TypeError: core_1.loadSchemaUsingLoaders is not a function
```

@ardatan 
Is it possible to see build connected to the PR so we will see build failures?

**Comment by ardatan** [comment]
@wtrocki After adding `integration` to `workspaces` in `package.json`, tests should work.

**Comment by wtrocki** [comment]
@ardatan Perfect. We can also remove then yarn install in CI as it will be pointless.

**Comment by wtrocki** [comment]
Done

**Comment by wtrocki** [comment]
Let's wait for build but it should be ok now.

**Comment by wtrocki** [comment]
This works perfectly on local machine using the same commands. Something strange happening with CircleCI

**Comment by wtrocki** [comment]
Update. it looks like circleci is optimizing dependencies in some place. Yarn graphql cannot be found.

**Comment by craicoverflow** [comment]
> Update. it looks like circleci is optimizing dependencies in some place. Yarn graphql cannot be found.

It looks like your node_modules are not getting installed at any stage during your test workflow, this is why `graphql` cannot be found.

I can replicate this exact behaviour when I remove `node_modukes` from the fullstack template.

Screenshot from 2020-01-15 08-54-54

**Comment by wtrocki** [comment]
Looks like the only thing that is missing is graphql cli.
Is not recognized as binary. 
Maybe the best way will be to call that out directly from the package folder

```
  Test folder0s
lrwxrwxrwx 1 runner docker   48 Jan 15 11:45 tsserver -> ../../../../node_modules/typescript/bin/tsserver
Run ls -al templates/fullstack/node_modules/.bin
total 8
drwxr-xr-x 2 runner docker 4096 Jan 15 11:45 .
drwxr-xr-x 3 runner docker 4096 Jan 15 11:45 ..
lrwxrwxrwx 1 runner docker   49 Jan 15 11:45 schemats -> ../../../../node_modules/schemats/bin/schemats.js
lrwxrwxrwx 1 runner docker   43 Jan 15 11:45 tsc -> ../../../../node_modules/typescript/bin/tsc
lrwxrwxrwx 1 runner docker   42 Jan 15 11:45 tslint -> ../../../../node_modules/tslint/bin/tslint
lrwxrwxrwx 1 runner docker   48 Jan 15 11:45 tsserver -> ../../../../node_modules/typescript/bin/tsserver
```

**Comment by wtrocki** [comment]
https://github.com/yarnpkg/yarn/issues/1585

**Comment by wtrocki** [comment]
@ardatan ready to be merged 👍

File: templates/fullstack/server/src/schema/generated.graphql [hunk 1]
```diff
@@ -11,7 +11,6 @@ type Comment {
   id: ID!
   title: String!
   description: String!
-  note: Note!
```
Comment (ardatan) [inline templates/fullstack/server/src/schema/generated.graphql]: Why don't we have relational fields anymore? @wtrocki

Comment (wtrocki) [reply templates/fullstack/server/src/schema/generated.graphql]: We moved to require this to be explicitly in model, but then our community did not like that so we have reverted this and going to restore it in the next release soon

**Pull request merged**